pkg/test/framework/components/csrctrl/README.md

```diff
@@ -0,0 +1,66 @@
+# CSR Controller
+
+This folder implements the CSR controller for Istio to sign Kubernetes CSR, please refer to [Kubernetes CSR API](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/) for more detail. BTW, this controler reuse many code from [signer-ca](https://github.com/cert-manager/signer-ca).
+
+## How to build image for CSR controller?
+
+CSR controller can be built into binary file named `csrctrl`, then built the CSR controller image based on this binary file as below:
+
+Generate the CSR controller binary file:
+
+```console
+$ make build csrctrl
+```
+
+Check the binary file under the folder of `istio/out/linux_amd64/`, then build the image based on it:
+
+```console
+$ make docker.csrctrl
+```
+
+User can find the CSR controller image by `docker images`, and see the $HUB/csrctrl with tag value $TAG.
+
+## How to deploy CSR controller in Kubernetes cluster in test code?
+
```

Comment:
Can we add  a section to notify users that it only supports signer from `istio.io/`?

Reply:
Sure